docs/tutorials/mongoid-upgrade.txt

```diff
@@ -362,6 +362,25 @@ Example Mongoid 7.1 behavior:
 Mongoid provides :ref:`sharding management Rake tasks <sharding-management>`
 to shard collections according to shard keys declared in models.
 
+Query Cache Moved to Driver
+------------------------------------
+
+Minor change: The Mongoid query cache has been deprecated in favor of a new
+and improved query cache implemented in Ruby driver version 2.14.0. Mongoid
+retains the legacy query cache for use with older versions of the driver, but
+when used with driver versions 2.14.0 and newer, Mongoid will use the driver
+query cache instead.
+
+The interface for enabling and disabling the query cache in Mongoid has not
+changed. When using driver versions 2.14.0 and newer, this interface will
+enable or disable the query cache in the driver.
+
+The driver query cache is more correct and more effective than the legacy
+Mongoid query cache. If you plan to use the query cache, it is recommended
+that you upgrade to driver version 2.14.
+
+To read more about the query cache improvements made in the driver, see
+`the Ruby driver documentation <https://docs.mongodb.com/ruby-driver/tutorials/query-cache/>`_.
```

Comment:
I believe this is a 404, I think https://docs.mongodb.com/ruby-driver/current/tutorials/query-cache/ would work.

Reply:
Yup, thank you!